src/AmqpBundle/DependencyInjection/Configuration.php

```diff
@@ -185,4 +151,43 @@ protected function addConsumers(ArrayNodeDefinition $node)
                 ->end()
             ->end();
     }
+
+    private function exchangeOptions()
```

Comment:
please add some phpdoc ? did you run cocke ? https://github.com/M6Web/AmqpBundle/blob/master/CONTRIBUTING.md . Thanks !